cinema_service/settings.py

```diff
@@ -45,6 +45,7 @@
     "rest_framework",
     "rest_framework.authtoken",
     "debug_toolbar",
+    "drf_spectacular",
```

Comment:
The 'drf_spectacular' package is correctly included in the INSTALLED_APPS list. Ensure that it is properly configured in the settings for API schema generation.